### PR TITLE
Add 2017 look back blog post

### PR DESCRIPTION
**Pull request by jbonofre**
NB: This PR is not yet ready to be merged: I have to add the image in the `images` folder.

However, it's ready for a review in term of content.

### CODE REVIEW

File: src/_posts/2018-01-09-beam-a-look-back.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+First, lets take a glimpse at how Beam was used in 2017. Apache Beam being a
+unified framework for batch and stream processing, enables a very wide spectrum
+of diverse use cases. Here are some use cases that exemplify the versatility of
+Beam.
+
+![Grid computer and numerical simulations, batch ETL and ML pre-processing,
+  streaming ETL, IoT and real-time anomaly detection, real-time mobile game
+  session personalization](
+  {{ "/images/beam_versatility_use_cases.svg" | prepend: site.baseurl }})
```
Comment (melap) [inline src/_posts/2018-01-09-beam-a-look-back.md]: I only just now notice that there's a blog directory inside images, I guess the image should go somewhere in there.

**Comment by jbonofre** [comment]
Just updated the PR to include image. Ready for a review ;)

File: src/_posts/2018-01-09-beam-a-look-back.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+<!--more-->
+
+## Use cases
+
+First, lets take a glimpse at how Beam was used in 2017. Apache Beam being a
+unified framework for batch and stream processing, enables a very wide spectrum
+of diverse use cases. Here are some use cases that exemplify the versatility of
+Beam.
+
+<img class="center-block"
```
Comment (melap) [inline src/_posts/2018-01-09-beam-a-look-back.md]: the jenkins verification job is failing due to image needing alt text:
image /images/blog/2017-look-back/timeline.png does not have an alt attribute (line 110)

I also think the image might need to be cropped to remove extra blank space around the graphic? from looking at staged version, there's a pretty big empty space below:
http://apache-beam-website-pull-requests.storage.googleapis.com/370/blog/2018/01/09/beam-a-look-back.html

**Comment by jbonofre** [comment]
I updated the blog post to include `alt` tag on `img`.

**Comment by jbonofre** [comment]
@melap can you take a look please ? Thanks !

**Comment by jbonofre** [comment]
@asfgit merge

**Comment by jbonofre** [comment]
Rebase/squash done.

**Comment by jbonofre** [comment]
@asfgit merge

**Pull request merged**